ytmusicapi/parsers.py

```diff
@@ -130,11 +124,20 @@ def parse_content_list(results, parse_func):
 
 
 def parse_album(result):
-    location = SUBTITLE if len(result['subtitle']['runs']) == 1 else SUBTITLE2
```

Comment:
`len(result['subtitle']['runs'])` is 1 for album without year, and it was detected as single with "Album" as publication year. I split this function into two to fix this.

Reply:
Good catch! I think I should include these cases in the tests or I'll keep missing them...